src/vsr.zig

```diff
@@ -651,6 +701,9 @@ pub const Header = extern struct {
         var header = Header{
             .cluster = cluster,
             .size = @sizeOf(Header),
+            // Note the root prepare is the only case where:
+            //   primary_index(prepare.view) ≠ prepare.replica
```

Comment:
obsolete comment?

Reply:
Ah, forgot to revert that second commit.